lib/common.js

```diff
@@ -45,6 +45,11 @@ module.exports.options = (yargs) => yargs
     alias: 'v',
     describe: 'Show verbose output',
     default: false,
+  }).option('revert', {
```

Comment:
shall we call it something like `remove-revert` to be more explicit? and perhaps rename the alias as `rr` or something

Reply:
I considered something around `remove` as well, that sounds perfect!